src/components/scenes/LoginScene.tsx

```diff
@@ -82,7 +82,7 @@ class LoginSceneComponent extends React.PureComponent<Props, State> {
       // @ts-expect-error
       .catch(e => this.setState({ backgroundImage: theme.backgroundImage }))
     const { YOLO_USERNAME, YOLO_PASSWORD, YOLO_PIN } = ENV
-    if (YOLO_USERNAME != null && (YOLO_PASSWORD != null || YOLO_PIN != null) && firstRun) {
+    if (YOLO_USERNAME != null && (Boolean(YOLO_PASSWORD) || Boolean(YOLO_PIN)) && firstRun) {
```

Comment:
Why not get the username too? Also, this is a pretty cool pattern we might use in more places.

Reply:
Fair point; Username could also be given this treatment. The type-casting pattern is nice, and I'm happy to promote it.